src/game/items/CItemMulti.cpp

```diff
@@ -2662,6 +2688,23 @@ bool CItemMulti::r_WriteVal(lpctstr ptcKey, CSString & sVal, CTextConsole * pSrc
             sVal.FormatVal(0);
             break;
         }
+        case SHL_REGION:
+        {
+            if (m_pRegion != nullptr)
+            {
+                if (!IsStrEmpty(ptcKey))
+                {
+                    if (m_pRegion)
```

Comment:
This is redundant, since the same check is done some line up

Reply:
Done, I don't know what I was thinking while coding it.